app-connector-client/server/routes/apis.js

```diff
@@ -39,49 +37,49 @@ function getAll(req, res) {
     }
 };
 
-function getAllAPIs(localKyma, cb) {
+function getAllAPIs(cb) {
     request({
-        url: CONFIG.URLs.metadataUrl,
+        url: connection.info().metadataUrl,
         method: "GET",
         agentOptions: {
-            cert: fs.readFileSync(certFile),
-            key: fs.readFileSync(keyFile)
+            cert: fs.readFileSync(connection.info().certificate),

```

Comment:
Instead of this, a `connection.getCertificate()` would be more object oriented and more autocomplete friendly.

Reply:
I changed it so that we keep the full file content of the key and certificate in memory and no one is required to read from file system. Furthermore, I changed it to connection.certificate() and connection.privateKey(). I think getters are not typical in javascript